### ISSUE TITLE
Example not working

### ISSUE THREAD

**Issue by plops** [NVIDIA/cutlass#88]
Hi,
I can't compile the _Launching a GEMM kernel in CUDA_ example that is given here:
https://github.com/NVIDIA/cutlass/blob/master/media/docs/quickstart.md

Typo cutlass instead:
`> line 2:   #include <**cutlas**/gemm/device/gemm.h>
`
D is not declared (perhaps use C?):
`> line 49:  int ldd = D.device_ref().stride(0);`

My own copy of this example:
https://github.com/plops/cl-cpp-generator2/blob/master/example/11_nvidia_cutlass/source/nvcut_00_cuda_main.cu

I'm on gentoo and try to compile with:

```
export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/martin/src/cutlass/build/tools/library/
export PATH=$PATH:/opt/cuda/nvvm/bin/
/opt/cuda/bin/nvcc ge.cu  -I /home/martin/src/cutlass/include/ -I /opt/cuda/include/ -I/home/martin/src/cutlass/tools/util/include/ -I/home/martin/src/tools/library/include -L/home/martin/src/cutlass/build/tools/library/ -lcutlass --std=c++14 -O1 -g -Xcompiler=-march=native --compiler-bindir=/usr/x86_64-pc-linux-gnu/gcc-bin/8.4.0 
```

**Comment by hwu36** [comment]
Thank you for pointing this out.  We will fix this in our next release.

**Comment by d-k-b** [comment]
Should be fixed in 2.3.
